src/style.css

```diff
@@ -1 +1,99 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
```

Comment:
The `src` attribute in the `@font-face` rule should use a relative path to the font file within the project directory. Ensure that the path is correct and accessible from the CSS file.